README.md

```diff
@@ -4,68 +4,73 @@ languages:
 - csharp
 products:
 - azure
+- azure-storage
+- dotnet-core
 description: "This repo contains sample code demonstrating the backup/restore and recoverable deletion functionality of Azure Key Vault using the Azure .NET SDK."
 urlFragment: net-sdk-samples
 ---
 
-# .NET SDK samples for recovering and restoring Azure Key Vault entities 
+# How to recover and restore Azure Key Vault entities with .NET 
 
-This repo contains sample code demonstrating the backup/restore and recoverable deletion functionality of Azure Key Vault using the [Azure .Net SDK](https://docs.microsoft.com/dotnet/api/overview/azure/key-vault?view=azure-dotnet). The scenarios covered by these samples include:
+## Prerequisites
 
-* Backing up and restoring Key Vault secrets and keys
-* Enabling recoverable deletion on creating a new vault
-* Enabling recoverable deletion on an existing vault
-* Recovering or permanently deleting deleted vaults
-* Recovering or permanently deleting Key Vault secrets, keys, and certificates
+To complete this tutorial:
 
-The recoverable deletion functionality is also referred to as 'soft delete'; consequently, a permanent, irrecoverable deletion is referred to as 'purge'.
+* Install .NET Core 3.1 version for [Linux] or [Windows]
```

Comment:
Use `-` consistently throughout the markdown (you use mainly this below).

Reply:
U mentioned steps need numbered, alternative should be bulleted. I think this is much better than mix them.
Do you still think we should keep consistency or changing below with numbered steps?
Like what i have done now.